packages/api/schema/mutation.js

```diff
@@ -111,7 +113,10 @@ export default [
       ): OrderItem!
 
       """
-      Change billing address and order contact of an open order (cart)
+      Change billing address and order contact of an open order (cart). All of the parameters
+      except order ID are optional and the update will ocure for parameters provided.
+      If the delivery provider or payment provider ID provided doesn’t already exist new
+      providers will be created with the provided ID.
```

Comment:
New providers will be created? Sure?

Reply:
@pozylon https://github.com/unchainedshop/unchained/blob/master/packages/core-orders/db/orders/helpers.js#L594

Reply:
https://github.com/unchainedshop/unchained/blob/master/packages/core-orders/db/orders/helpers.js#L580

Reply:
Please fix this to "new order payments will be created", providers are never automatically created through a mutation.

Reply:
**Fixed**